ndex2/client.py

```diff
@@ -691,8 +907,26 @@ def _make_network_public_indexed(self, network_id):
 
     def make_network_private(self, network_id):
         """
+
         Makes the network specified by the network_id private.
 
+        .. code-block:: python
+
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		network_id = "INSERT NETWORK UUID HERE"
+		my_ndex.make_network_private(network_id)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client